whatwg.org/service-worker.js

```diff
@@ -1,10 +1,7 @@
 "use strict";
 
-self.cacheKey = "v1";
+self.cacheKey = "v2";
```

Comment:
It seems you could remove this given the default, but maybe it's better if there's no default and each of these service-worker.js files have it explicitly so we don't forget about it when we update those files?

Reply:
I think upon reflection the better thing to do is concatenate them. We want to have the cache refresh if the base service worker list of resources/fetching logic changes, or if the individual list changes. Let me do that instead.